subsys/bluetooth/host/mesh/net.c

```diff
@@ -1358,12 +1334,29 @@ void bt_mesh_net_recv(struct net_buf_simple *data, s8_t rssi,
 	}
 }
 
-static void ivu_complete(struct k_work *work)
+static void ivu_refresh(struct k_work *work)
 {
-	BT_DBG("");
+	bt_mesh.ivu_duration += BT_MESH_IVU_HOURS;
+
+	BT_DBG("%s for %u hour%s",
+	       bt_mesh.iv_update ? "IVU in Progress" : "IVU Normal mode",
+	       bt_mesh.ivu_duration, bt_mesh.ivu_duration == 1 ? "" : "s");
+
+	if (bt_mesh.ivu_duration < 96) {
```

Comment:
96 appears a couple of times, maybe worth a macro

Reply:
> 96 appears a couple of times, maybe worth a macro

@carlescufi sure. I've updated the PR with a `BT_MESH_IVU_MIN_HOURS` macro for it.